edx_proctoring/templates/proctored_exam/instructions.html

```diff
@@ -79,12 +79,12 @@ <h2 id="acessible-error-title">{% trans "Cannot Start Proctored Exam" %}</h2>
 </div>
 
 <div class="footer-sequence border-b-0 padding-b-0">
-  {% if not is_sample_attempt %}
-  <p class="proctored-exam-instruction">
+  {% if not is_sample_attempt and allow_unproctored %}
+    <p class="proctored-exam-instruction">
       {% trans "Don't want to take this exam with online proctoring?" %}<a class="proctored-decline-exam" href='#' data-action="decline" data-exam-id="{{exam_id}}" data-change-state-url="{{change_state_url}}">
         {% trans "Take this exam as an open exam instead." %}
```

Comment:
@andy-armstrong @robrap How about:
"Take this exam without proctoring. Doing so means that you are no longer eligible for academic credit." 
where "Take this exam without proctoring" is the link text.